pages/Messages/Overview.md

```diff
@@ -13,26 +13,57 @@ menu: Messages
     * Public messages (profile changes, public posts, reactions)
     * Direct messages between accounts (dead drop, inbox)
     * Key changes (private graph KeyList, encryption KeyList, public key rotation)
     * Graph changes (follow/unfollow)
-1. Specify an on-chain message format
+1. Specify an on-chain announcement format
 1. Provide data size estimations
 1. Facilitate use of SDK and interpretation of on-chain data
 
 ## Assumptions
-* Chain messages are on Ethereum
-* Message data is posted via [Ethereum log events](https://medium.com/mycrypto/understanding-event-logs-on-the-ethereum-blockchain-f4ae7ba50378)
-* Signature algorithm is [secp256k1](https://en.bitcoin.it/wiki/Secp256k1). This allows the use of `ecreover`
-  to get public keys. A public key also need not be included in a log event for ease of validation.
-* content hashes are created via the same [keccak-256 hashing algorithm](https://en.wikipedia.org/wiki/SHA-3) used by Ethereum.
 
-## DSNP Message Formats
-We have seriously considered two possibilities, a [variable message format](#Variable-Message-Format), and a [unified message format](#unified-message-format).  Others are listed at the end of this page.
+* Announcements are on Ethereum.
+* Announcement data is posted via [Ethereum log events](https://medium.com/mycrypto/understanding-event-logs-on-the-ethereum-blockchain-f4ae7ba50378).
+* Signature algorithm is [secp256k1](https://en.bitcoin.it/wiki/Secp256k1). This allows the use of `ecreover` to get public keys. A public key also need not be included in a log event for ease of validation.
+* Content hashes are created via the same [keccak-256 hashing algorithm](https://en.wikipedia.org/wiki/SHA-3) used by Ethereum.
+
+## Terminology
+
+In the interest of clarity, we will define the terms "message," "content," "announcement" and "event" as distinct but related objects going forward in this specification.
+
+#### Announcement
+
+An announcement refers specifically to a DSNP item as described in this specification document.
+An announcement is intended for inclusion in a batch file and to eventually be posted to the blockchain via a batch log event.
+Some announcements, such as broadcasts and replies, will contain references to content items.
+
+#### Content
+
+Content items, or simply content, refers to an activity pub compliant object hosted at some URI and intended to be posted via an announcement in a batch.
+Generally, content within the DSNP specification will be defined by the [W3C Activity Pub specification](https://www.w3.org/TR/activitypub/#object-without-create), however extensions, such as the [Mastadon Activity Pub specification](https://docs.joinmastodon.org/spec/activitypub/) may also be implemented.
+This specification will not directly define any standards for content.
```

Comment:
Hmmm... This feels a bit wrong as just above this we specify it as matching the W3C Activity Pub. I think we do need to "define" the standards, just not in this page (which might be actually what "this spec" means.)

Reply:
Removed line.